libs/ui/src/search_select.tsx

```diff
@@ -0,0 +1,342 @@
+import React, { useRef, useState } from 'react';
+import styled from 'styled-components';
+import { assert, assertDefined } from '@votingworks/basics';
+import { Icons } from './icons';
+
+function noop() {
+  // do nothing
+}
+
+export interface SelectOption<T extends string = string> {
+  value: T;
+  label: string;
+}
+
+const ComponentContainer = styled.div`
+  width: 100%;
+  position: relative;
+`;
+
+const SelectBar = styled.div`
+  width: 100%;
+  min-height: 2.2rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border: ${(p) => p.theme.sizes.bordersRem.medium}rem solid
+    ${(p) => p.theme.colors.foreground};
+  border-radius: 0.25rem;
+  background: ${(p) => p.theme.colors.background};
+`;
+
+const ValuesContainer = styled.div<{
+  isMulti: boolean;
+}>`
+  display: ${(p) => (p.isMulti ? 'flex' : 'grid')};
+  flex: 1 1 0%;
+  flex-wrap: wrap;
+  align-items: center;
+  gap: 0.1rem;
+  margin: 0.2rem 0.3rem;
+  border-radius: 0.25rem;
+  overflow: hidden;
+`;
+
+const INPUT_CONTAINER_GRID_AREA = '1 / 1 / 2 / 3';
+const INPUT_GRID_AREA = '1 / 2 / auto / auto';
+
+const SelectedSingleValue = styled.div`
+  grid-area: ${INPUT_CONTAINER_GRID_AREA};
+  max-width: 100%;
+  font-weight: ${(p) => p.theme.sizes.fontWeight.semiBold};
+`;
+
+const SelectedMultiValue = styled.div`
+  height: 100%;
+  display: inline-flex;
+  background: ${(p) => p.theme.colors.foreground};
+  color: ${(p) => p.theme.colors.background};
+  border-radius: 0.25rem;
+`;
+
+const SelectedMultiValueText = styled.div`
+  padding: ${(p) => {
+    const thickRem = p.theme.sizes.bordersRem.thick;
+    return `${thickRem}rem ${thickRem}rem ${thickRem}rem ${thickRem * 2}rem`;
+  }};
+  white-space: nowrap;
+  font-weight: ${(p) => p.theme.sizes.fontWeight.semiBold};
+`;
+
+const SelectedMultiValueRemove = styled.div`
```

Comment:
we should probably use `styled.button` here, since it's playing the role of a button (or at least assign it a `role="button"` attribute and give it a `tabindex="0"` so it's properly accessible)

Reply:
handled by `react-select`